app/serializers/user_serializer.rb

```diff
@@ -287,6 +289,14 @@ def primary_group_flair_color
     object.try(:primary_group).try(:flair_color)
   end
 
+  def external_id
+    object.try(:single_sign_on_record).try(:external_id)
```

Comment:
`try` is slower than `&.` :grin:

Reply:
Thanks! I've updated it. I was using `try` because that's what is used in the other methods in the class.